cli/change-detection/index.md

```diff
@@ -21,6 +21,39 @@ E.g., to list all stacks that contain changes:
 terramate list --changed
 ```
 
+Since `v0.11.0` untracked and uncommitted files are also considered changes.
+
+::: info
+_Untracked file_ means Git doesn't know about its existence.
+
+_Uncommitted file_ means Git knows about them but the latest changes in them are not committed to Git yet.
```

Comment:
```suggestion
_Uncommitted file_ is a file where changes are recognized by Git but not yet committed.
```

Reply:
this is not much accurate either. Check my other answer.